bin/buildBlueprints.js

```diff
@@ -130,40 +130,56 @@ console.log('...Reading Blueprints', argv.blueprintsPath);
 console.log('...cwd', process.cwd());
 
 var config = makeConfig(argv);
+var isTest = argv.target === TARGETS.TEST;
 
 build(config, function(stats) {
   if (stats.errors && stats.errors.length > 0 && !argv.watch) {
     console.log(colors.red('ERROR IN BUILD. Aborting.'));
     process.exit(1);
   }
 
-  if (argv.runTest) {
+  if (isTest) {
     console.log(colors.magenta(
       '\n   ******************************' +
       '\n   *       RUNNING TESTS        *' +
       '\n   ******************************'
     ));
 
-    m = new Mocha({ reporter: mochaNotifier.decorate('spec') });
-    stats.assets.forEach(function(asset) {
-      m.addFile('./.test/' + asset.name);
+    var testDirectories = config.builds.map(function(build) {
+      return build.webpackConfig.output.path;
```

Comment:
These might need a prefix of `'./'` here -- when I install this patch via git url pointing to commit 00484fb0c6a96ee542d20ebd69e1e599c33d2f4f, I can find all the tests in `reddit-mobile@2X`, but none of them run. `glob` seems to be defined but it's not finding anything (see my comment about `glob`'s placement in `package.json`, but manually installing `7.0.5` didn't seem to fix the problem either). Is there something special I need to do? I'll post a link to an updated version of 2X that uses this new branch for testing purposes